docs/source/performance.rst

```diff
@@ -0,0 +1,94 @@
+Performance Benchmarks
+======================
+
+Integrated Benchmarks
+---------------------
+
+DGL continuously evaluates the speed of its core APIs, kernels as well as the training speed
+of the state-of-the-art GNN models. The benchmark code is available at
+`the main repository <https://github.com/dmlc/dgl/tree/master/benchmarks>`_. They are triggered
+for every nightly-built version and the results are published to
+`https://asv.dgl.ai/ <https://asv.dgl.ai>`_.
+
+v0.6 Benchmarks
+---------------
+
+To understand the performance gain of DGL v0.6, we re-evaluated it on the v0.5 benchmarks
+plus some new ones for graph classification tasks against the updated baselines. The results
+are available in `a standalone repository <https://github.com/dglai/dgl-0.5-benchmark>`_.
+
+v0.5 Benchmarks
```

Comment:
I think the following table shows the performance of dgl 0.4.x instead of dgl 0.5

Reply:
I remember the number is for 0.5. The paper footnote also points to the 0.5-benchmark repo.